massa-pool/src/tests/scenario.rs

```diff
@@ -183,6 +184,172 @@ async fn test_pool() {
     )
     .await;
 }
+
+#[tokio::test]
+#[serial]
+async fn test_pool_with_execute_sc() {
+    let (cfg, thread_count, operation_validity_periods, max_pool_size_per_thread): &(
+        PoolSettings,
+        u8,
+        u64,
+        u64,
+    ) = &POOL_SETTINGS;
+
+    pool_test(
+        cfg,
+        *thread_count,
+        *operation_validity_periods,
+        async move |mut protocol_controller, mut pool_command_sender, pool_manager| {
+            let op_filter = |cmd| match cmd {
+                cmd @ ProtocolCommand::PropagateOperations(_) => Some(cmd),
+                _ => None,
+            };
+            // generate transactions
```

Comment:
please move this comment to just above the loop, and add more description(for example why the duplication?)